core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -750,6 +751,17 @@ public ClientFactoryBuilder useHttp2Preface(boolean useHttp2Preface) {
         return this;
     }
 
+    /**
+     * Sets whether to use HTTP/1.1 instead of HTTP/2. If enabled, the client will not attempt to upgrade to
+     * HTTP/2 for {@link SessionProtocol#HTTP} and {@link SessionProtocol#HTTPS}.
+     * This option is disabled by default.
+     */
+    @UnstableApi
+    public ClientFactoryBuilder preferHttp1(boolean preferHttp1) {
```

Comment:
How about `useHttp1`?
For me, this sounds like the client will try with HTTP/2 if the server doesn't support HTTP/1.1

Reply:
`useHttp1` sounds like the client does not allow/use HTTP/2. As HTTP/2 is still used for `H2(C)`, a weaker expression was picked.

Reply:
That makes sense.
Then, how about adding
`The client will use HTTP/2 if {@link SessionProtocol#H2} or {@link SessionProtocol#H2C} is used.`

Reply:
Good idea.